examples/stein_dmm.py

```diff
@@ -0,0 +1,340 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Example: Deep Markov Model inferred using SteinVI
+=================================================
```

Comment:
Where do I add `stein_dmm.png` to make it a thumbnail?

Reply:
I think you put it into the right place.